setup.cfg

```diff
@@ -25,6 +25,7 @@ test = pytest
 
 [tool:pytest]
 collect_ignore = ['setup.py']
+xfail_strict=true
```

Comment:
what does this flag do?

Reply:
It makes it so that any test marked as expected fail, if that test succeeds, will be marked as a failure. Basically protects against xfails tests becoming outdated imo.